core/transport/src/proto/journal/journal.proto

```diff
@@ -24,6 +24,7 @@ message JournalEntry {
   optional AsyncPersistRequestEntry async_persist_request = 16;
   optional BlockContainerIdGeneratorEntry block_container_id_generator = 3;
   optional BlockInfoEntry block_info = 4;
+  optional string cluster_id = 42;
```

Comment:
should this be something like a "ClusterInfo" type, which contains the string `cluster_id` and potentially other fields in the future, like start time or something?

Reply:
sgtm